src/Modules/Content/Dnn.PersonaBar.Pages.Tests/PagesControllerUnitTests.cs

```diff
@@ -23,13 +24,16 @@ public class PagesControllerUnitTests
         Mock<IUrlRewriterUtilsWrapper> _urlRewriterUtilsWrapperMock;
         Mock<IFriendlyUrlWrapper> _friendlyUrlWrapperMock;
         Mock<IContentVerifier> _contentVerifierMock;
-
         PagesControllerImpl _pagesController;
+        Mock<IPortalController> _portalControllerMock;
+
+        TabInfo _tab = new TabInfo();
+        PortalSettings _portalSettings = new PortalSettings();
```

Comment:
I think I's prefer to keep `TabInfo` and `PortalSettings in individual test methods